.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,67 @@
+name: Bug 🐛
+description: Something is not behaving as expected.
+title: '[Bug] '
+labels: ['bug']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Before creating your issue
+  - type: markdown
+    attributes:
+      value: |
+        ## Troubleshoot locally
+        Please try to ensure that the bug you're reporting is _not_ caused by any misconfiguration on your machine or network. Ideally, try to reproduce the problem on a different environment.
+  - type: markdown
+    attributes:
+      value: |
+        ## Identify the steps to reproduce the issue
+        The fewer and simpler the steps we can take to reproduce your issue, the sooner we can get back to you, and solve your problem.
+        If possible, please include a GIF or screenshots for each of the steps you followed to reproduce the bug.
+        Please provide as much relevant information as possible about your environment if you think that could be interfering with the app's behavior.
+  - type: textarea
+    id: bug_summary
+    attributes:
+      label: Bug summary
+      description: A concise description of the bug.
+      placeholder: The recorder crashed when I started recording.
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: Recorder Version
+      description: |
+        Which version of the script recorder are you using?
+        If you're unsure, provide us with the date in which you've downloaded it (if you remember it).
```

Comment:
Maybe instead we could provide them instructions on how to see the version? The production-built version of the recorder does include the version name.

Reply:
Great idea, just added it!